web/src/admin/applications/wizard/commit/ak-application-wizard-commit-application.ts

```diff
@@ -134,10 +138,25 @@ export class ApplicationWizardCommitApplication extends BasePanel {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             .catch(async (resolution: any) => {
                 const errors = await parseAPIError(resolution);
+
+                // THIS is a really gross special case; if the user is duplicating the name of an
+                // existing provider, the error appears on the `app` (!) error object. We have to
+                // move that to the `provider.name` error field so it shows up in the right place.
+                if (isValidationError(errors) && Array.isArray(errors?.app?.provider)) {
+                    const providerError = errors.app.provider;
+                    errors.provider = errors.provider ?? {};
+                    errors.provider.name = providerError;
+                    delete errors.app.provider;
+                    if (Object.keys(errors.app).length === 0) {
+                        delete errors.app;
+                    }
+                }
+
+                this.errors = errors;
```

Comment:
This is the dumbth that caused the errors not to be preserved locally and so weren't shown locally.